textdb/textdb-dataflow/pom.xml

```diff
@@ -41,6 +41,21 @@
             <artifactId>textdb-storage</artifactId>
             <version>${textdb.version}</version>
         </dependency>
+
+
+        <dependency>
+            <groupId>edu.stanford.nlp</groupId>
+            <artifactId>stanford-corenlp</artifactId>
+            <version>3.6.0</version>
+        </dependency>
+        <dependency>
+            <groupId>edu.stanford.nlp</groupId>
+            <artifactId>stanford-corenlp</artifactId>
+            <version>3.6.0</version>
+            <classifier>models</classifier>
+        </dependency>
+
+
         <dependency>
             <groupId>org.mockito</groupId>
```

Comment:
Based on our discussion on Monday, I am not sure whether Mockito is really needed for your test case.  @sandeepreddy602 Thoughts?